metacat-common-server/src/main/java/com/netflix/metacat/common/server/connectors/exception/InvalidTableObjectException.java

```diff
@@ -0,0 +1,63 @@
+/*
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.netflix.metacat.common.server.connectors.exception;
+
+import com.netflix.metacat.common.QualifiedName;
+
+import javax.annotation.Nullable;
+
+/**
+ * InvalidObjectException.
+ *
+ * @author zhenl
+ * @since 1.2.0
+ */
+public class InvalidTableObjectException extends ConnectorException {
```

Comment:
We can get rid of this class. We use InvalidMetaException for any invalid meta info.